Tests/L0/ServiceFabricUpdateAppVersions/SimpleSuffix.ps1

```diff
@@ -3,46 +3,60 @@ param()
 
 . $PSScriptRoot\..\..\lib\Initialize-Test.ps1
 
+$taskPath = "$PSScriptRoot\..\..\..\Tasks\ServiceFabricUpdateAppVersions"
+
+Microsoft.PowerShell.Core\Import-Module $taskPath\Assert-SingleItem.psm1
+Microsoft.PowerShell.Core\Import-Module $taskPath\Update-ServiceVersions.psm1
+Microsoft.PowerShell.Core\Import-Module $taskPath\Update-PackageVersion.psm1
```

Comment:
Why is it necessary to import these from the test when it's already being done as a result of dot sourcing the Update-ApplicationVersions.ps1 script?